docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -1,3 +1,13 @@
+= Deprecation of 'legacy' cookies
+
+Since the introduction of the https://web.dev/articles/samesite-cookies-explained[SameSite attribute] on cookies provided by Keycloak, another set of 'legacy' cookies (name suffixed in `_LEGACY`) were also introduced in order to support older browsers that are not compatible with this attribute. Since the `SameSite` attribute is now https://caniuse.com/same-site-cookie-attribute[widely supported], this behavior is now deprecated and will be removed in the next release, so that only the cookies using the `SameSite` attribute remain.
+
+You can restore this behavior (for now) by enabling the `legacy-cookies` feature. For example:
+
+```
+kc.[sh|bat] start-dev --features legacy-cookies
```

Comment:
`start-dev` is for developers while release notes target non-dev users. Perhaps better to use `start` command here.

Reply:
Should be resolved now :+1: